dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -186,6 +186,20 @@ private bool AreScrollButtonsVisible()
             }
         }
 
+        private bool IsScrollIncreaseButtonEnabled()
```

Comment:
>IsScrollIncreaseButtonEnabled [](start = 21, length = 29)

these appear to be unused

Reply:
I am still working on the UI tests, this will be needed later. I will remove "WIP" in the PR title once its done.